go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/gorilla/websocket v1.4.2 // indirect
 	github.com/graphql-go/graphql v0.7.9 // indirect
 	github.com/kylelemons/godebug v1.1.0 // indirect
```

Comment:
That would mean this PR is the right place, so if we roll this back, it takes out the go mod changes (since this PR introduces the three imports)? If that's what you mean - I'm all for it.